test/e2e/set/scyllacluster/config.go

```diff
@@ -6,9 +6,9 @@ import (
 
 const (
 	updateFromScyllaVersion  = "5.2.6"
```

Comment:
`updateFromScyllaVersion` and `upgradeFromScyllaVersion` could be bumped to newer versions as well